code/modules/ruins/lavaland_ruin_code.dm

```diff
@@ -125,14 +125,15 @@
 	radio = /obj/item/device/radio/headset/syndicate/alt
 	back = /obj/item/weapon/storage/backpack
 	pocket1 = /obj/item/weapon/gun/ballistic/automatic/pistol
+	pocket2 = /obj/item/weapon/card/id/syndicate/anyone
 	roundstart = FALSE
 	death = FALSE
 	icon = 'icons/obj/Cryogenic2.dmi'
 	icon_state = "sleeper"
 	has_id = 1
```

Comment:
remove this and the id_access_list, otherwise people will spawn with both IDs(one that works and one that doesn't).